src/BookCard/BookCard.css

```diff
@@ -0,0 +1,124 @@
+.book {
+  width: 200px;
+  height: 100px;
+  /* margin: 5% auto; */
+  background: #fff;
+  perspective: 70em;
+  position: relative;
+  box-shadow: 0 0 1px 1px rgba(0, 0, 0, 0.5), 0 0 0 3px #fff, 0 0 2px 4px rgba(0, 0, 0, 0.5), 0 0 0 7px #9388BB, 0 0 1px 8px rgba(0, 0, 0, 0.5), 0 0 30px rgba(0, 0, 0, 0.51);
+  overflow: hidden;
+  margin: 1rem;
+
+}
+
+.page {
+  background: #fff;
+  width: 50%;
+  height: 100%;
+  float: right;
+  transform-origin: left;
+  transform: rotateY(0deg);
+  opacity: 1;
+  transition: all 5s ease-in-out;
+  animation: turnpage 20s infinite linear;
+  overflow: hidden;
+
+}
+
+.pagenext {
```

Comment:
Make sure to use kebab case for class names 🍢

Reply:
That's a great point.  Will need to refactor.